module/purpose/Soccer/data/config/webots/Soccer.yaml

```diff
@@ -6,4 +6,4 @@ force_playing: false
 
 # The position of the robot
 # GOALIE, STRIKER, DEFENDER
-position: STRIKER
+position: DEFENDER
```

Comment:
revert, you should only set defender for webots2 and webots3 (and nugus2 and nugus3)